app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -54,7 +53,7 @@ public function create(): View
      */
     public function store(RegisterRequest $request): RedirectResponse
     {
-        $user = User::create([
+        $user = AuthorizedUser::create([
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar `$authorizedUser` . 


````suggestion
        $authorizedUser= AuthorizedUser::create([